openshift/e2e-common.sh

```diff
@@ -193,6 +193,7 @@ function install_catalogsource(){
   git clone --depth 1 https://github.com/openshift-knative/serverless-operator.git ${SERVERLESS_DIR}
   pushd ${SERVERLESS_DIR}
 
+  git checkout release-1.20
```

Comment:
Let's add this to the clone command via `--branch` like in https://github.com/openshift/knative-serving/commit/532a1db4cd8e5c6c016956c0de6fb0fc4e62f1c6